src/db/collections/block.rs

```diff
@@ -94,6 +94,16 @@ impl BlockCollection {
         Ok(())
     }
 
+    // /// Gets the length of the collection.
```

Comment:
Remove?

Reply:
Yes, please remove this and rebase. It would be great to get this merged before the end of this sprint.